examples/text-generation/README.md

```diff
@@ -21,6 +21,23 @@ Conditional text generation on Intel® Gaudi® AI Accelerators. You can find mor
 
 ## Requirements
 
+Please make sure to follow [Driver Installation](https://docs.habana.ai/en/latest/Installation_Guide/Driver_Installation.html) to install Gaudi driver on the system.
+We suggest to use pytorch docker image to run below examples.
+
+To use dockerfile provided for the sample, please follow [Docker Installation](https://docs.habana.ai/en/latest/Installation_Guide/Additional_Installation/Docker_Installation.html) to setup habana runtime for Docker images.  
+The docker image helps users to setup pytorch software and packages to run the samples. Users still need to install required packages like deepspeed to run the samples.  
+
+### Docker Run
+After docker build, users could follow below command to run and docker instance and users will be in the docker instance under text-generation folder.
+```bash
+docker run -it --runtime=habana -e HABANA_VISIBLE_DEVICES=all -e OMPI_MCA_btl_vader_single_copy_mechanism=none   --cap-add=ALL --privileged=true  --net=host --ipc=host  -v "$PWD":/workspace --workdir  /workspace  vault.habana.ai/gaudi-docker/1.19.1/ubuntu24.04/habanalabs/pytorch-installer-2.5.1:latest
+```
```

Comment:
why do we need any docker file here instead of gaudi doc?

Reply:
@libinta  you are right. remove all docker contents here. only keep instruction updates.